flight/PiOS/Common/pios_streamfs.c

```diff
@@ -542,7 +542,7 @@ int32_t PIOS_STREAMFS_Init(uintptr_t *fs_id, const struct streamfs_cfg *cfg, enu
 		goto out_exit;
 	}
 
-	streamfs->com_buffer = (uint8_t *)PIOS_malloc(cfg->write_size);
```

Comment:
This specific malloc looks like it would result in breakage if this filesystem is ever used on top of a flash driver that uses DMA (ie. where ```PIOS_FLASH_write_data()``` uses DMA underneath.